huaweicloud/services/acceptance/fgs/resource_huaweicloud_fgs_function_test.go

```diff
@@ -285,13 +299,48 @@ func TestAccFgsV2Function_logConfig(t *testing.T) {
 	})
 }
 
-func testAccFgsV2Function_basic(rName string) string {
+func zipFileUploadResourcesConfig() string {
+	randName := acceptance.RandomAccResourceNameWithDash()
+
+	return fmt.Sprintf(`
+variable "script_content" {
+  type    = string
+  default = <<EOT
+def main():  
+    print("Hello, World!")  
+
+if __name__ == "__main__":  
+    main()
+EOT
+}
+
+resource "huaweicloud_obs_bucket" "test" {
+  bucket = "%[1]s"
+  acl    = "private"
+
+  provisioner "local-exec" {
+    command = "echo '${var.script_content}' >> test.py\nzip -r test.zip test.py"
+  }
+  provisioner "local-exec" {
+    command = "rm test.zip test.py"
+    when    = destroy
+  }
+}
+
+resource "huaweicloud_obs_bucket_object" "test" {
+  bucket = huaweicloud_obs_bucket.test.bucket
+  key    = "test.zip"
+  source = "${abspath("./")}/test.zip"
```

Comment:
using abspath directly will be fine

```suggestion
  source = abspath("./test.zip")
```

Reply:
done.